keen/__init__.py

```diff
@@ -441,3 +441,125 @@ def get_all_collections():
     """
     _initialize_client_from_environment()
     return _client.get_all_collections()
+
+def create_access_key(name, is_active=True, permitted=[], options={}):
```

Comment:
Was there a deliberate decision made to put this right in `__init__.py` instead of doing something liked the Saved/Cached Query Interface? Just curious.

Reply:
The answer to all of these was, for the most part, to avoid deviating from the existing patterns I saw.   The other answer is that I wanted to create the simplest path for users to run Access Key operations, which meant hiding the client (shoving it in `__init__` like the basic queries) but also allowing them to work with a specific client in case they have multiple projects they are running in a single Python process.

To be honest though, I made that decision pretty lightly and could be persuaded to move it into its own interface. I don't like how monolithic that file has become.

Reply:
I'm ambivalent, so whatever feels right to you.